### ISSUE TITLE
Add option to use a folder as asset

### ISSUE THREAD

**Issue by Razican** [mmstick/cargo-deb#5]
Currently, if a folder wants to be added as an asset, this is not possible:

``` toml
assets = [
    ["vendor/*", "var/lib/app/vendor/", "755"],
]
```

There is no option to put all the files in one folder in another folder, and we need to copy all files in that directory one by one. Could this be fixed? It would be great to be able to use wildcards at least.

**Comment by mmstick** [comment]
It could easily be fixed, and pull requests are welcome if you have ideas on how to improve functionality. I'm working on a lot of other Rust-related projects right now.

**Comment by Razican** [comment]
OK, I didn't have time to check at the code (I have never created a `.deb` file myself, so I'll try to learn about it. Thanks!

In any case, I guess that the package is still not available in crates.io, right?

**Comment by mmstick** [comment]
It won't be available in Crates.io until the tar crate officially supports creating a hidden `.` directory.

**Comment by messense** [comment]
@Razican I have implemented glob pattern assets in #37
